src/components/buttons.js

```diff
@@ -36,34 +37,18 @@ import { SvgIcon } from './SvgIcon';
  * @param {ButtonProps} props
  */
 function ButtonBase({
-  children,
   className,
   icon,
   iconPosition = 'left',
-  disabled,
-  expanded,
-  pressed,
-  onClick = () => {},
   size = 'medium',
-  style = {},
-  title,
   variant = 'normal',
+  expanded,
+  pressed,
+  ...restProps
 }) {
-  const otherAttributes = {};
-  if (typeof disabled === 'boolean') {
-    otherAttributes.disabled = disabled;
-  }
-  if (typeof title !== 'undefined') {
-    otherAttributes.title = title;
-    otherAttributes['aria-label'] = title;
-  }
-
-  if (typeof expanded === 'boolean') {
-    otherAttributes['aria-expanded'] = expanded;
-  }
-  if (typeof pressed === 'boolean') {
-    otherAttributes['aria-pressed'] = pressed;
-  }
+  restProps['aria-expanded'] = expanded;
```

Comment:
If `expanded` is undefined the prop is ignored

Reply:
Thank you; much simpler.